Makefile

```diff
@@ -135,7 +135,7 @@ check-generated-sources-up-to-date: regenerate-sources
 pkg/addons/default/assets.go: pkg/addons/default/assets/*
 	env GOBIN=$(GOBIN) time go generate ./$(@D)
 
-pkg	/nodebootstrap/assets.go: pkg/nodebootstrap/assets/*
```

Comment:
I believe this was fixed in master already.

Reply:
doesn't seem like it